src/fontra/views/editor/editor.html

```diff
@@ -5,6 +5,8 @@
     <title>Fontra</title>
     <link href="/css/core.css" rel="stylesheet" />
     <link href="/editor/editor.css" rel="stylesheet" />
+    <script defer src="/third-party/lib-font/inflate.js"></script>
```

Comment:
Can we import these in panel-reference-font.js? They are quit out of place here.

Reply:
I did, it looks clearer.